src/components/views/SuggestedViews/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useQuery } from 'react-query';
+import { useRouter } from 'next/router';
+import { Grid, makeStyles, Theme } from '@material-ui/core';
+
+import { getSuggestedViews } from 'utils/datasetUtils';
+import getViews from 'fetching/views/getViews';
+import { useUser } from 'hooks';
+import ViewCard from './ViewCard';
+import ZetkinQuery from 'components/ZetkinQuery';
+import ZetkinSection from 'components/ZetkinSection';
+import { ZetkinView } from 'types/zetkin';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+        [theme.breakpoints.up('sm')]: {
+            flexDirection: 'row',
+        },
+        marginBottom: theme.spacing(1),
+    },
+    item: {
+        flex: 1,
+    },
+}));
+
+const SuggestedViews: React.FunctionComponent = () => {
+    const classes = useStyles();
+    const router = useRouter();
+    const user = useUser();
+    const { orgId } = router.query;
+    const viewsQuery = useQuery(['views', orgId], getViews(orgId as string));
+    const onClickCard = (evt: React.ChangeEvent<HTMLButtonElement>) => {
+        router.push(`/organize/${orgId}/people/views/${evt.currentTarget.value}`);
+    };
+
+    // if fewer than 3 views supplied, render nothing
+    if (!viewsQuery.data || viewsQuery.data.length < 3) return <div />;
```

Comment:
I think the logic here is sound (good catch!) but I'm not a fan of the single-line if statement or the "div as nothing". Is it not possible to return `null` from a `FunctionComponent`?